packet/src/test/java/org/jclouds/packet/compute/internal/BasePacketApiMockTest.java

```diff
@@ -130,15 +131,15 @@ protected RecordedRequest assertSent(MockWebServer server, String method, String
       RecordedRequest request = server.takeRequest();
       assertEquals(request.getMethod(), method);
       assertEquals(request.getPath(), path);
-      assertEquals(request.getHeader("Accept"), "application/json");
+      assertEquals(request.getHeader("Accept"), "application/json; version=" + ctx.getMetadata().get("apiVersion"));
       assertEquals(request.getHeader("X-Auth-Token"), X_AUTHORIZATION_TOKEN);
       return request;
    }
 
    protected RecordedRequest assertSent(MockWebServer server, String method, String path, String json)
          throws InterruptedException {
       RecordedRequest request = assertSent(server, method, path);
-      assertEquals(request.getHeader("Content-Type"), "application/json");
+      assertEquals(request.getHeader("Accept"), "application/json; version=" + ctx.getMetadata().get("apiVersion"));
```

Comment:
This is already checked in the method called in the line above. This one is intended to verify requests with a body and thus it checks they have the right content-type header.